packages/nodes-base/nodes/Webhook.node.ts

```diff
@@ -155,7 +155,7 @@ export class Webhook implements INodeType {
 				description: 'The HTTP Response code to return',
 			},
 			{
-				displayName: 'Respond when',
+				displayName: 'Respond When',
```

Comment:
You should update this in Wait node as well.. though I would prefer to keep this small letter

Reply:
I don't disagree, but it's the only one for the entire node using sentence case instead of capital case so it just looks strange

Reply:
got it